napari/layers/shapes/_shape_list.py

```diff
@@ -353,6 +353,178 @@ def add(
             # Set z_order
             self._update_z_order()
 
+    def add_multiple(
+        self,
+        shapes,
+        face_colors=None,
+        edge_colors=None,
```

Comment:
These default values seem like they would cause an error in the for loop on line 399.

Reply:
Correct, thanks for catching!